docs/api/api_blueprint.apib

```diff
@@ -4330,10 +4274,19 @@ Create a new tax using an event_id.
             {
               "data": {
                 "attributes": {
-                  "tax-name": "Service Tax",
-                  "tax-rate": "12",
-                  "tax-id": "1"
-                },
+                    "city": "example",
+                    "zip": "123456",
+                    "country": "IN",
```

Comment:
@shubham-padia Should we add validation for `PAYMENT_COUNTRIES` like in event invoice API [here](https://github.com/fossasia/open-event-orga-server/blob/nextgen/app/api/event_invoices.py#L34)?
Rest lgtm.